Code/GraphMol/FMCS/Wrap/rdFMCS.cpp

```diff
@@ -130,10 +132,9 @@ struct PyAtomBondCompData {
   MCSBondCompareFunction standardBondTyperFunc;
 };
 
-struct PyCompareFunctionUserData {
+struct PyCompareFunctionData : MCSCompareFunctionsData {
```

Comment:
With the restructuring of userData, I think this kind of extension is more logical.